### ISSUE TITLE
Conflict with 'statsd' install dir

### ISSUE THREAD

**Issue by kaidokert** [wolph/python-statsd#42]
pip install statsd python-statsd 
results in a mess in site-packages. Unlikely to be a real issue, only noticed because i was trying to figure out the differences between these and pystatsd, statsd-client

**Comment by WoLpH** [comment]
Perhaps I should add some detection to see if the conflicting version is installed. It's a bit unfortunate that @jsocol and I chose the same name when starting the packages but nearly impossible to change that now since many projects depend on either of the packages.

One option could be to create a compatibility layer so they function with the same API. Although I would argue that my API is currently more Pythonic. To illustrate:
```
>>> import statsd
>>> client = statsd.Client('foo')
>>> counter = client.get_counter('bar')
>>> counter += 10
```
Versus:
```
>>> import statsd
>>> client = statsd.StatsClient(prefix='foo')
>>> client.incr('bar', 10)
```

Or the more common case:
```
>>> import statsd
>>> counter = statsd.Counter('foo')
>>> counter += 10
```
Versus:
```
>>> import statsd
>>> client = statsd.StatsClient()
>>> client.incr('foo', 10)
```

The examples for timers and all are similar.

**Comment by jsocol** [comment]
Honestly, it never occurred to me that someone would install both in the same virtualenv.

**Comment by kaidokert** [comment]
Yeah i didn't plan on installing them side by side either, so its a very minor thing. Like i said, i only noticed because i was looking at various statsd client options available.

**Comment by WoLpH** [comment]
So far in the years I've maintained this package you're the 2nd to report this so you're definitely not alone :)

Perhaps I'll add some API compatibility and/or checks, should be good

**Comment by wolph** [comment]
Documented in the README as of v3.0.0, which is out now.

This package predates jsocol's `statsd` and both install a module called `statsd`, so renaming either import would break every existing user. The conflict is documented rather than fixed: install exactly one of the two.

The README carries the note under "Note on the package name", and the docs are at https://python-statsd.readthedocs.io/
